kibana-reports/public/components/report_definitions/edit/edit_report_definition.tsx

```diff
@@ -51,6 +52,7 @@ export function EditReportDefinition(props) {
   };
 
   const reportDefinitionId = props['match']['params']['reportDefinitionId'];
+  let curReportDefinition: ReportDefinitionSchemaType;
```

Comment:
np: can we spell out the entire word -> `currentReportDefinition` vs `curReportDefinition`

Reply:
will change